tools/autodeploy/templates/ambiqsuite_example/Makefile.template

```diff
@@ -1,4 +1,13 @@
-rootdir := ..
+# Initialize rootdir to be cwd
```

Comment:
I see where you're going with this, but this won't work. This makefile is intended to live inside an ambiqsuite example (e.g. boards/apollo4p/examples/ai/kws/gcc). That being said, this functionality is completely broken right now for other reasons, so leave this in and we'll fix it when we fix the rest of the ambiqsuite functionality.